### PR TITLE
Fix the build issues for STM32Fxx (#433)

### PR DESCRIPTION
**Pull request by ugo1983**
This was reported here - https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/issues/411

Signed-off-by: Gaurav Aggarwal <aggarg@amazon.com>

Co-authored-by: Aniruddha Kanhere <60444055+AniruddhaKanhere@users.noreply.github.com>

Description
-----------

Test Steps
-----------

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/portable/NetworkInterface/STM32Fxx/stm32fxx_hal_eth.h [hunk 1]
```diff
@@ -45,11 +45,11 @@
     #define __STM32F7xx_HAL_ETH_H
 
     #if defined( STM32F7xx )
-        #include "stm32f7xx_hal_def.h"
+        #include "stm32f7xx_hal.h"
```
Comment (htibosch) [inline source/portable/NetworkInterface/STM32Fxx/stm32fxx_hal_eth.h:48]: /md
Indeed, that change is necessary, if not there will be compiler errors like `#error please define ETH_RX_BUF_SIZE`

**Review by htibosch — APPROVED** [review]
Thank you @aggarg for this PR. I had stumbled on the same problem.
I approve it.

**Comment by htibosch** [comment]
Hello @ugo1983, any news on this PR?

It would be nice if it can be merged because new PR's for `ipv6_multi` are queuing up on my laptop:-)

File: source/portable/NetworkInterface/STM32Fxx/stm32fxx_hal_eth.h (hunk 1 above)
Comment (htibosch) [reply source/portable/NetworkInterface/STM32Fxx/stm32fxx_hal_eth.h:48]: Ping ping

**Review by aggarg — APPROVED** [review]

**Review by AniruddhaKanhere — APPROVED** [review]

**Pull request merged**